src/v2/v2_types.rs

```diff
@@ -0,0 +1,185 @@
+use chrono::TimeZone;
```

Comment:
I don't think we should have a v2_types. It's just makign it harder to discover. We have `types.rs` already.